src/common/mfu_flist_walk.c

```diff
@@ -488,7 +488,7 @@ static void walk_stat_process_dir(char* dir, CIRCLE_handle* handle)
                 }
                 else {
                     /* name is too long */
-                    MFU_LOG(MFU_LOG_ERR, "Path name is too long: %lu chars exceeds limit %lu", len, sizeof(newpath));
+                    MFU_LOG(MFU_LOG_ERR, "Path name is too long: %lu chars exceeds limit %lu, (%s/%s)", len, sizeof(newpath), dir, name);
```

Comment:
This is good, but let's tweak the format just a bit.  I try to use a pseudo-convention to always contain printed path names in single quotes set off by a colon.
```suggestion
                    MFU_LOG(MFU_LOG_ERR, "Path name is too long, %lu chars exceeds limit %lu: '%s/%s'", len, sizeof(newpath), dir, name);
```

Reply:
@adammoody  done.